Dockerfile.bundled

```diff
@@ -0,0 +1,16 @@
+FROM debian:buster
+
+RUN set -eux \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get upgrade;apt-get update \
```

Comment:
Not an apt expert by any means, but shouldn't `update` go before `upgrade`?

Reply:
Good note @DolphinChips , I will edit the commit and merge it.